app/models/blueprint.rb

```diff
@@ -0,0 +1,51 @@
+class Blueprint < ApplicationRecord
+  has_many :service_templates
+  private  :service_templates, :service_templates=
```

Comment:
`service_templates` are all service templates that are contained in the blueprint. Do we really need to expose them without maintaining the hierarchical relationships? I think what we really need is only the top level template which is all the method `bundle` is about.